.circleci/config.yml

```diff
@@ -307,11 +307,11 @@ workflows:
         - build_and_test
     - hold_uat:
         type: approval
-        requires:
-        - build_and_test
     - deploy_uat:
         requires:
         - hold_uat
+        - build_and_test
+        - run_webhint
```

Comment:
Essentially yes, but also, it isn't useful to deploy to UAT unless Webhint passes.  

I appreciate that people might not want to wait for Webhint to pass before authorising UAT (which is why I brought the hold UAT forward), but if Webhint doesn't pass, it shouldn't be sent to UAT.  

That was my thinking anyway.

Reply:
_"it isn't useful to deploy to UAT unless Webhint passes"_
I don't agree with this. The webhint job could return errors that are invisible from the UAT tester.
At the moment, `deploy_uat` is after `build_and_test` only because  `deploy_uat` requires the docker image built by `build_and_test`

Reply:
Okay, I'll change this.